src/menus.js

```diff
@@ -0,0 +1,111 @@
+const {app, Menu, shell} = require('electron');
+const packageJson = require('../package.json');
+
+const RELOAD_WEBVIEW = {
+  label: 'Reload',
+  accelerator: 'CmdOrCtrl+R',
+  click(_, focusedWindow) {
+    if (focusedWindow) {
+      focusedWindow.webContents.send('reload-requested');
+    }
+  }
+};
+
+const TOGGLE_WEBVIEW_DEVTOOLS = {
+  label: 'Toggle Developer Tools',
```

Comment:
Are we going to ever have localization for these, or is that a non-goal?

Reply:
To my knowledge that's currently out-of-scope.  If CSD ever gets localized I suppose we'd want to do these too.